src/main/java/green/projectile/ProjectileGraph.java

```diff
@@ -0,0 +1,44 @@
+package green.projectile;
+import javax.swing.*;
+import java.awt.*;
+
+public class ProjectileGraph extends JComponent {
+
+    private Projectile projectile = new Projectile(0,0);
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        g.translate(0, getHeight());
+
+        g.setColor(Color.BLACK);
+
+        double totalTime = projectile.getTotalFlightTime();
+        projectile.setSeconds(0);
+
+        for(double i = 0; i <= totalTime; i += .01) {
+            double currX = projectile.getX();
+            double currY = projectile.getY();
+
+            projectile.setSeconds(i);
+
+            g.drawLine((int)currX, -(int)currY, (int)projectile.getX(), -(int)projectile.getY());
+        }
+
+        g.setColor(Color.BLUE);
+
+        double peakY = projectile.getPeakY();
+        projectile.setSeconds(totalTime / 2);
+        double xMid = projectile.getX();
```

Comment:
Can you also use getY here?

Reply:
I would find the seconds at peak y value when going through it on lines 20-27, save it, and use that to find x later -- is that worth the extra overhead?

Reply:
What I mean is that you're setting the seconds on the projectile and getting the X value. You can also get the Y value instead of calling getPeakY